spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "menu"
+
+describe Menu do
+
+  let(:menu){ {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.